src/js/structs/__tests__/Job-test.js

```diff
@@ -35,13 +35,37 @@ describe('Job', function () {
   describe('#getDescription', function () {
 
     it('returns the description', function () {
-      let job = new Job({id: '/foo', description: 'bar'});
+      let job = new Job({id: 'foo', description: 'bar'});
 
       expect(job.getDescription()).toEqual('bar');
     });
 
   });
 
+  describe('#getDocker', function () {
+
+    it('returns the docker configuration', function () {
+      let job = new Job({id: 'foo', run: {docker: {image: 'busybox'}}});
+
+      expect(job.getDocker()).toEqual({image: 'busybox'});
+    });
+
+    it('defaults to an empty object if property is undefined', function () {
```

Comment:
It reduces the number of checks needed when accessing properties (e.g. `job.getDocker().image`)